common/oidc-client/oauth2-proxy/base/kustomization.yaml

```diff
@@ -1,42 +1,58 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
+namespace: oauth2-proxy
```

Comment:
@kimwnasptd before it was in the istio-system namespace and dex is still in the auth namespace if i remeber correctly

Reply:
Yes, we want to integrate with existing oauth deployments in enterprise environments